doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -5,7 +5,6 @@ top_table:
   Zigbee: ZIGBEE
   Thread: NET_L2_OPENTHREAD
   LTE: LTE_LINK_CONTROL
-  HomeKit: HOMEKIT
```

Comment:
Hi @gautesl 
This doesn't seem to remove HomeKit entry from the top table. Should the `:remove-rows:` option be used to achieve that? https://github.com/nrfconnect/sdk-nrf/blob/main/doc/_extensions/software_maturity_table.py#L33-L37
Or is there some other way?

Reply:
That should not be necessary, this change will affect the report that is used when building the sml part of the docs, so you will not be able to see this change before after the PR is merged